pkg/pillar/cmd/zedrouter/dnsmasq.go

```diff
@@ -161,9 +161,10 @@ func createDnsmasqConfiglet(
 		file.WriteString("no-resolv\n")
 	}
 
-	for _, ipset := range ipsets {
+	for _, host := range ipsets {
```

Comment:
The old code has a confusing name ipsets; it isn't the name of the sets but the corresponding host/domainnames without the ipv4./ipv6. prefix. So probably makes sense to rename the ipsets variable/arg for clarity. ipsethosts?

Reply:
Agree regarding confusions (was confused myself :))
I would suggest following naming scheme:
`ipsetName = ipv[46].<ipsetBasename>`
where `<ipsetBasename>` can be a special value (such as `local`) or it can be hostname as is the case for ACLs with `host` match. Wherever `ipset string` actually refers to a hostname used with ipset, I would call it `ipsetHost string` (and `ipsetHosts` for an array of those).
I have already done these naming changes in `ipset.go` and in the code touched by this PR (see latest changes).
What requires further renaming are these functions: https://github.com/lf-edge/eve/blob/master/pkg/pillar/cmd/zedrouter/acl.go#L106-L229
However, that code requires also some refactoring to make it less confusing and also to fix the issue that I have mention in the Slack. I will prepare it either as a separate commit or as a separate PR altogether.

Reply:
That naming makes sense to me.